oio/api/object_storage.py

```diff
@@ -705,6 +706,7 @@ def object_create_ext(self, account, container, file_or_path=None,
         """
         if (data, file_or_path) == (None, None):
             raise exc.MissingData()
+        kwargs['reqid'] = request_id('create-')
```

Comment:
Unfortunately this overwrites the request ID set by the client application or the method decorator.

Reply:
Done